concrete/config/concrete.php

```diff
@@ -892,6 +892,10 @@
          * Enable asciify to sanitize name of uploaded files
          */
         'enable_filename_asciify' => true,
+        /**
```

Comment:
This is a normal comment, not a phpdoc comment, right?
If so, what about using `/*` instead of `/**` ?